### DISCUSSION TITLE
number of CPU used when training with GPU

### DISCUSSION THREAD

**Discussion by Lewis-YL** [deepmodeling/deepmd-kit#1818] [Q&A] [answered]
Hi,

In the deepmd v2.1.3, 

Before dp train input.json, I used the commands:
export OMP_NUM_THREADS=1
export TF_INTRA_OP_PARALLELISM_THREADS=1
export TF_INTER_OP_PARALLELISM_THREADS=1

However, It still shows a log like this:
![image](https://user-images.githubusercontent.com/38740371/178073860-91cbbd89-65b2-4408-9ff6-ae077f9c3063.png)

This did not happen in the previous deepmd version.
When I used the other GPU or other CPU, it slows my current training speed by 25% percent.
![image](https://user-images.githubusercontent.com/38740371/178074668-915e53b8-8ea1-4e47-896b-7b449bbc0266.png)

Do you know how to fix it?

**Comment by njzjz** [comment, marked as answer]
Your log only shows one thread:

> `OMP: Info #254: KMPAFFINITY: pid 212948 tid 213000 thread 0 bound to os proc set 0`

It looks expected. Did you show the entire OMP log?

**Reply by Lewis-YL** [reply]
Yes, that is the entire OMP log.
I want to use 1 CPU core of my 24 cores.

You can see the training time changed from 60s to 73s because I was running another job using a different GPU card.
In principle, this should not affect my current training job.

Do you know what is the possible cause?

I have not seen this in previous deepmd version.

**Reply by njzjz** [reply]
The log of another job will be helpful. Do they bound to the same proc set?
